frame_id: "map"

body:
  frame_id: "base_link"

timer_main:
  rate: 10.0 # [Hz]

# Basketball
#bounded_box:
#  min_x: -6.0 # [m]
#  max_x: 6.0 # [m]
#  min_y: -6.0 # [m]
#  max_y: 6.0 # [m]
#  min_z: 2.0 # [m]
#  max_z: 8.0 # [m]
  
# IST Lamp
bounded_box:
  min_x: -1.0 # [m]
  max_x: 11.0 # [m]
  min_y: -6.0 # [m]
  max_y: 6.0 # [m]
  min_z: 3.0 # [m]
  max_z: 9.0 # [m]

rrt:
  N_max: 20
  N_termination: 200
  N_yaw_samples: 10
  radius: 2.0 # [m]
  step_size: 2.0 # [m]
  tolerance: 1.0 # [m]

camera:
  frame_id: "camera_link"
  h_fov: 1.51844 # [rad]
  width: 1080
  height: 720
  min_distance: 0.2 # [m]
  max_distance: 5 # [m]

path:
  uav_radius: 2.0 # [m]
  lambda: 0.5 # (tunning factor)
  
